gtsam/navigation/ImuBias.h

```diff
@@ -142,10 +142,10 @@ class GTSAM_EXPORT ConstantBias {
     return q - (*this);
   }
   Vector6 GTSAM_DEPRECATED localCoordinates(const ConstantBias& q) {
-    return between(q).vector();
+    return (q - (*this)).vector();
```

Comment:
Is this inadvertently added?

Reply:
This is fine. It is just addressing the GTSAM deprecation in ImuBias.h.